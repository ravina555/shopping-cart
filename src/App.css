.Products{
  display:inline-block

}

.shoppingCardsWrapper{
  width:30%;
  margin:25px;
  background-color:gainsboro!important;
  display:inline-block!important
}

.Summary{
  margin:20px;
  width:97%!important
}