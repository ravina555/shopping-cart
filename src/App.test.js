import App from './App';
import Enzyme,{shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() })
describe('ToggleComponent', () => {

  it('Should render App component', () => {
     let appInstance = shallow(<App/>);
     const tree = renderer.create(<App/>).toJSON();
     expect(appInstance.props().className).toBe('App')
     expect(appInstance.props().children).toBeDefined();

     expect(tree).toMatchSnapshot();
    
  });
});
