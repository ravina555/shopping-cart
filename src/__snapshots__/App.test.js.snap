// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToggleComponent Should render App component 1`] = `
<div
  className="App"
>
  <div
    className="cartBg"
  >
    <div
      className="Products"
    />
  </div>
</div>
`;
